### ISSUE TITLE
Process to follow after revoking a certificate.

### ISSUE THREAD

**Issue by plutocrat** [acmesh-official/acme.sh#2887]
Hi, more of a clarification than an issue, although it might make a useful addition to the documentation. 

A client recently had their webserver private key exposed. It probably wasn't intercepted, but the correct thing to do is to revoke it and issue a new one. I have found the revoke command. 
acme.sh --revoke  -d example.com   --revoke-reason 1

After this, what is the correct acme procedure to make sure the private.key is replaced with a new one. A normal --renew will keep the same private.key. Will --issue create a new private.key? Or should I completely remove the domain with all config and then start again? Or something else ...

**Comment by plutocrat** [comment]
OK, well I just went ahead and tried this out. I've no idea if it was the correct way to do things. 
- I revoked the old cert
- I went into the acme.sh/domain.com/ directory and deleted all the certs and csr files, but left the conf file
- I ran --issue to re-generate the certs. 
- I ran --renew to install them into my webserver ssl dir. 
That seemed to work. 
Just deleting the private.key file didn't work as it wasn't re-generated. It seemed to need the csr files deleting to generate a new private.key. 
Hope this helps someone
